Pages/LocalEvents.razor

```diff
@@ -5,6 +5,7 @@
 @attribute [Microsoft.AspNetCore.Authorization.AllowAnonymous]
 @using Microsoft.Extensions.Configuration;
 @inject IConfiguration m_configuration
+@inject IHttpClientFactory client
```

Comment:
I would call this variable m_httpClientFactory(easier to understand later)

Reply:
Renamed